Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -108,12 +108,16 @@ - (void)viewDidLoad
 }
 
 - (void)setOptions:(WPMediaPickerOptions *)options {
+    WPMediaPickerOptions *originalOptions = _options;
     _options = [options copy];
+    BOOL refreshNeeded = (originalOptions.filter != options.filter) | (originalOptions.showMostRecentFirst != options.showMostRecentFirst) | (originalOptions.allowCaptureOfMedia != options.allowCaptureOfMedia);
```

Comment:
Some of the options don't need to force a refresh, like allowsMultipleSelection.

On the main app on the MediaLibrary if I didn't implement it like this you will see a "flash", when switching from Edit Mode to View Mode.